azure/test/lint.test.js

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2018 VMware, Inc.
+SPDX-License-Identifier: MIT
+*/
+
+const nock = require('nock');
+const { lintTestEnv } = require('./helper.test');
+
+const {
+  gzipLogs,
+  gunzipData,
+  sendHttpRequest,
+  LIntKafkaEnv,
+  flattenJson,
+} = require('../src/lint');
+
+describe('flattenJson', () => {
+  it('should flatten arrays in JSON', () => {
+    const exampleJson = ['a', 'b', 'c'];
+    const result = flattenJson(exampleJson);
+    expect(result).toMatchSnapshot();
+  });
+
+  it('should work when there is nothing to flatten', () => {
+    const exampleJson = {
+      field1: 'value1',
+      field2: 'value2',
+    };
+    const result = flattenJson(exampleJson);
+    expect(result).toMatchSnapshot();
+  });
+
+  it('should flatten nested arrays and objects', () => {
+    const exampleJson = {
+      field1: { field11: 'value11', field12: 'value12' },
+      field2: 'value2',
+      field3: 3,
+      field4: { field41: ['a', 'b', 'c'] },
+    };
+    const result = flattenJson(exampleJson);
+    expect(result).toMatchSnapshot();
+  });
+});
+
+describe('LIntHttpEnv', () => {
+  it('should create options for the HTTP stream', () => {
+    const STRUCTURE = 'cloudwatch';
```

Comment:
In Azure, we are seeing test cases for AWS. Can you please check it out

Reply:
Removed aws test cases